src/commands/start.ts

```diff
@@ -18,24 +18,27 @@ export default class Start extends Command {
   async run() {
     const PWD = process.env.PWD ?? ''
     const userConfig = await getLocalConfig()
-    const WIKI_GIT_FOLDER = join(PWD, userConfig.localWikiFolder, 'iota-wiki')
+    const WIKI_GIT_FOLDER = join(PWD, userConfig.localWikiFolder ?? '', userConfig.localWikiFolder ? 'iota-wiki' : '')
```

Comment:
maybe we should rename `WIKI_GIT_FOLDER` to `WORKING_FOLDER` or something like that.

Reply:
Sounds good. But then I will also rename `localWikiFolder` to `workingFolder`

Reply:
nah, would leave that. the config actually specifies the the local wiki folder